### DISCUSSION TITLE
Why ?

### DISCUSSION THREAD

**Discussion by SIMONHSCH** [3kh0/ext-remover#1110] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Why does bookmarkets executor is not on [Chrose](https://chrose.netlify.app/?) anymore ? because it works for me.

**Comment by AshtonDavies** [comment, marked as answer]
I removed it because it is useless for Extension Launcher. @akabutnicer would probably agree, as he discourages even the bookmarklet, which I am pretty sure would not be necessary in any block screen situation.

I suggest we just forget about all of it and move on.

**Reply by AshtonDavies** [reply]
@SIMONHSCH Also, wdym it works for you?

**Reply by SIMONHSCH** [reply]
yes
